make/generate.mk

```diff
@@ -0,0 +1,7 @@
+.PHONY: generate-kubefed-crd
+generate-kubefed-crd: vendor
```

Comment:
I'm wondering - cannot that be part of the `api` repo? In the same way as is done for host/member-operator repos. We do have the dependency to KubeFed project as well, so it should be doable...

Reply:
Makes sense.
Moved to API - https://github.com/codeready-toolchain/api/pull/57